tools/batch-script/address_provider.go

```diff
@@ -39,7 +39,7 @@ type AddressProvider struct {
 	currentIndex     uint
 }
 
-const endOfAccountsError = "storage used is not initialized or not initialized correctly"
+const endOfAccountsError = "get storage used failed"
```

Comment:
the wording here is kinda awkward

```suggestion
const endOfAccountsError = "failed to get storage used"
```

alternatively:
```suggestion
const endOfAccountsError = "aw heck we did a lil failure while getting storage used >_< gomenasorry"
```

Reply:
Just to clarify, this error is produced by flow-go / FVM and we are searching for it, not producing it.